spec/models/event_spec.rb

```diff
@@ -111,7 +118,23 @@
       end
     end
   end
-  
+
+  context 'when destroyed' do
```

Comment:
This has been added in a previous patch with:

`it { is_expected.to have_many(:submissions).dependent(:destroy) }`